### ISSUE TITLE
Nested !> does not work

### ISSUE THREAD

**Issue by Thinkscape** [jsoverson/preprocess#19]
For example
````html
  <!-- @if true !>
  <title>
  <!-- @echo TITLE !>
  </title>
  <!-- @endif -->
```

This will compile to:
````html
  <title>
  <!-- @echo TITLE !>
  </title>
````
----

If I change
````
  <!-- @echo TITLE !>
````

to 
````
  <!-- @echo TITLE -->
````

.... the interpolation will work but it will result in malformed HTML.

**Comment by anseki** [comment]
Hi, thank you for your great program.

Now, did you not change your policy?
I think that supporting nested directives is very important.
For example, there are many platforms, and those are different, and more new platforms (e.g. Windows 10) are coming.
We must specify those by multiple parameters.

```html
foo bar
foo bar

<!-- @ifdef DEBUG -->
<div id="debug-info">
  foo bar
  foo bar

  <!-- @if platform === 'ios' -->
  <div>something for iOS</div>
  <!-- @endif -->

  foo bar
  foo bar

  <!-- @if platform === 'android' -->
  <div>
    something for Android

    <!-- @if version < 4 -->
    <div>something for old</div><!-- Only Android even if old version -->
    <!-- @endif -->

  </div>
  <!-- @endif -->

  foo bar
  foo bar

  <!-- @if platform === 'win' -->
  <div>
    something for Windows

    <!-- @ifndef TOUCH_DEVICE -->
    <div>something for non-touch-device</div>
    <!-- @endif -->

  </div>
  <!-- @endif -->

  foo bar
  foo bar
</div>
<!-- @endif -->

foo bar
foo bar
```

**Comment by BendingBender** [comment]
This was fixed in #61. Bear with me, I'm preparing a new release. In the meantime, you can use the master branch which already contains this functionality.

**Comment by anseki** [comment]
Ok, Thanks!
